### DISCUSSION TITLE
Product Images Are Not Published

### DISCUSSION THREAD

**Discussion by tonyoconnell** [swellstores/community#119] [Help] [answered]
Hello. I am using Origin V 10 hosted on Swell. When we create a new product the images don't appear on the production site, but they do on the dev site. The thumbnails appear on both sites. To get around this I am able to push from my local machine and it rebuilds the site and the images appear after the site is built. Obviously this is not ideal since we want to allow someone to be able to add a  product without a developer. What should we do?

**Comment by logeshswell** [comment, marked as answer]
@tonyoconnell In production site, static build is generated for faster performance while in dev site, dynamic (or server) build is generated.
Hence, you need to rebuild the site by triggering the deployment if you make any changes in the product or other settings.
